LEGO1/viewmanager/viewlod.h

```diff
@@ -28,19 +28,19 @@ class ViewLOD : public LODObject {
 
 	Tgl::MeshBuilder* GetMeshBuilder() { return m_meshBuilder; }
 	const Tgl::MeshBuilder* GetMeshBuilder() const { return m_meshBuilder; }
-	undefined4 GetUnknown0x08() { return m_unk0x08; }
-	unsigned char GetUnknown0x08Test4() { return m_unk0x08 & 0xffffff04; }
-	unsigned char GetUnknown0x08Test8() { return m_unk0x08 & 0xffffff08; }
+	int GetFlags() { return m_flags; }
+	unsigned char ContainsOnlyFlags() { return m_flags & 0xffffff04; }
```

Comment:
What does `ContainsOnlyFlags` mean?

Reply:
To be honest I'm not sure regarding the logic, but when this "flag" is set it skips reading the rest of the data. So in the file it only contains the flags.

Maybe `SkipReadingData` is better?

Reply:
Yeah, I think that would be better